src/platform/platform_linux.cpp

```diff
@@ -166,3 +166,15 @@ bool mp::platform::is_image_url_supported()
 {
     return true;
 }
+
+mp::NetworkInterfaceInfo mp::platform::get_network_interface_info(const std::string& iface_name)
+{
+    // TODO
+    return mp::NetworkInterfaceInfo{iface_name, "", "", mp::nullopt};
+}
+
+std::map<std::string, mp::NetworkInterfaceInfo> mp::platform::get_network_interfaces_info()
+{
+    // TODO
+    return std::map<std::string, struct mp::NetworkInterfaceInfo>();
+}
```

Comment:
I think these should throw instead, especially the first one, to avoid producing meaningless data. I understand you are concerned with testing vbox on linux, but we can checkout the other branch for that. In parallel, I think that other branch should be stacked on this one, but I'll better explain in that PR.

Reply:
Yeah, right. We discussed about throwing and I forgot it :(